doc/mivot.tex

```diff
@@ -282,13 +283,15 @@ \section{Client APIs}
 \input{api.tex}
 
 \section{Changes from Previous Versions}
-No previous versions yet.  
+MIVOT has been published as IVOA WD in April 2022 (\url{https://ivoa.net/documents/MIVOT/20220407/index.html}).
+Since that time, most of the changes were both in the text and the client code rather than in the syntax. 
+The latest can however be traced from GitHub.
```

Comment:
Boo!  At least try to enumerate changes reviewers and, more importantly, implementors should be aware of.  As long as we have the crazily long lines in this source code, I find the diffs hard to read, and I certainly won't want to trace changes from them once I have a (partial) implementation...

Reply:
most of the syntax changes (but one) were done before the WD publication